src/easylink/cli.py

```diff
@@ -10,6 +10,63 @@
     handle_exceptions,
 )
 
+SHARED_OPTIONS = [
+    click.option(
+        "-p",
+        "--pipeline-specification",
+        required=True,
+        type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+        help="The path to the pipeline specification yaml file.",
+    ),
+    click.option(
+        "-i",
+        "--input-data",
+        required=True,
+        type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+        help="The path to the input data specification yaml file (not the paths to the input data themselves).",
+    ),
+    click.option(
+        "-o",
+        "--output-dir",
+        type=click.Path(exists=False, dir_okay=True, resolve_path=True),
+        help=(
+            "The directory to write results and incidental files (logs, etc.) to. "
+            "If no value is passed, results will be written to a 'results/' directory "
+            "in the current working directory."
+        ),
+    ),
+    click.option(
+        "--timestamp/--no-timestamp",
+        default=True,
+        show_default=True,
+        help="Save the results in a timestamped sub-directory of --output-dir.",
+    ),
+    click.option(
+        "-e",
+        "--computing-environment",
+        default=None,
+        show_default=True,
+        type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+        help=("Path to a computing environment yaml file on which to launch the step."),
+    ),
+    click.option(
+        "-v", "--verbose", count=True, help="Increase logging verbosity.", hidden=True
+    ),
+    click.option(
+        "--pdb",
+        "with_debugger",
+        is_flag=True,
+        help="Drop into python debugger if an error occurs.",
+        hidden=True,
+    ),
```

Comment:
I think this is fine for now, but in the future I'd like to not use these for the DAG command, as they don't really make sense. I understand you currently have it saving out to the same sort of output path as the pipeline itself, which means some of these are needed.

Reply:
I'm going to drop computing_environment, verbose, and pdb, and leave "--timestamp".